x/shield/types/types.go

```diff
@@ -69,20 +69,25 @@ func NewPendingPayouts(amount sdk.Dec, to string) PendingPayout {
 // Provider tracks A or C's total delegation, total collateral,
 // and rewards.
 type Provider struct {
-	Address          sdk.AccAddress `json:"address" yaml:"address"`
-	DelegationBonded sdk.Coins      `json:"delegation_bonded" yaml:"delegation_bonded"`
-	Collateral       sdk.Coins      `json:"collateral" yaml:"collateral"`
-	TotalLocked      sdk.Coins      `json:"total_locked" yaml:"total_locked"`
-	Available        sdk.Int        `json:"available" yaml:"available"`
-	Rewards          MixedDecCoins  `json:"rewards" yaml:"rewards"`
+	// address of the provider
+	Address sdk.AccAddress `json:"address" yaml:"address"`
+	// bonded delegations
+	DelegationBonded sdk.Coins `json:"delegation_bonded" yaml:"delegation_bonded"`
+	// collateral, including that in withdrawal queue and excluding that being locked
+	Collateral sdk.Coins `json:"collateral" yaml:"collateral"`
```

Comment:
Not really related to this PR, but I think collateral related stuff should be tracked using `sdk.Coin` or `sdk.Int` like how delegations are dealt with in x/staking because we are only allowing bond denom. What do you think?

Reply:
I agree. Initial thinking was we need to be as general as possible with Coins, but I don't think that's really necessary in foreseeable future. I'd suggest we use `sdk.Int` instead.

Reply:
Cool. I’ll make the change.